### PR TITLE
Text/misca

### PR DESCRIPTION
**Pull request by Maascha**

### CODE REVIEW

File: src/lang/introduction/de.tsx [hunk 1]
```diff
@@ -25,7 +25,7 @@ const introduction = {
     },
     groupDescription: {
         first: 'In den Nachhilfe-Kursen hilfst du Schüler:innen, die sich in einem bestimmten schulischen Thema verbessern wollen. Besonders gefragt sind Themen aus den Hauptfächern (Deutsch, Mathe, Englisch). Die Kurse finden in der Regel wöchentlich für max. 12 Wochen statt.',
-        second: 'In den Deutsch-Kursen kannst du geflüchteten Schüler:innen beim Lernen der deutschen Sprache helfen. Wir stellen dir Unterrichtsmaterialien zur Verfügung, damit du Kurse gestalten kannst, in denen Sprechen und Üben im Vordergrund steht. Die Kurse finden wöchentlich statt.',
+        second: 'Bei Deutsch für Anfänger kannst du geflüchteten Schüler:innen beim Lernen der deutschen Sprache helfen. Wir stellen dir Unterrichtsmaterialien zur Verfügung, damit du Kurse gestalten kannst, in denen Sprechen und Üben im Vordergrund steht. Die Kurse finden wöchentlich statt.',
```
Comment (Jonasdoubleyou) [inline src/lang/introduction/de.tsx:28]: (Bei "Deutsch für Anfänger"?)

File: src/modals/DissolveMatchModal.tsx [hunk 2]
```diff
@@ -55,11 +57,11 @@ const DissolveMatchModal: React.FC<DissolveModalProps> = ({ showDissolveModal, o
                 )}
                 {currentIndex === 1 && (
                     <>
-                        <Modal.Header>Neue Lernpartner:in anfordern</Modal.Header>
+                        (DEACTIVATE_PUPIL_MATCH_REQUESTS === 'false' &&
```
Comment (Jonasdoubleyou) [inline src/modals/DissolveMatchModal.tsx:60]: Also for students?

Comment (Maascha) [reply src/modals/DissolveMatchModal.tsx]: good catch

**Comment by Maascha** [comment]
Also fixes project-user #704

**Review by Jonasdoubleyou — APPROVED** [review]

**Pull request merged**